rotkehlchen/api/rest.py

```diff
@@ -1025,6 +1025,7 @@ def add_history_event(self, event: HistoryBaseEntry) -> Response:
                     event,
                     mapping_values={
                         HISTORY_MAPPING_KEY_STATE: HISTORY_MAPPING_STATE_CUSTOMIZED,
+                        # TODO: pass the correct chain id from frontend
```

Comment:
Why to do? Add this here too and make sure it's exposed in the API and we let frontend know they need to do that.

Reply:
Just wanted to keep PRs smaller, but will add it now